pkg/ipmi/ipmi.go

```diff
@@ -173,3 +207,47 @@ func (i *IPMI) ShutoffWatchdog() error {
 
 	return nil
 }
+
+func (e *Event) marshall() []byte {
+	buf := &bytes.Buffer{}
+
+	if err := binary.Write(buf, binary.LittleEndian, *e); err != nil {
+		log.Fatalf("Can't convert struct %v to bytes", *e)
```

Comment:
I don't think we want to log from the library, return an error if needed.

Reply:
Updated it.

Reply:
I don't think we should print/log messages in the pkg/library.